### PR TITLE
Adding file permissions for ConcentricAI integration.

### PR DESCRIPTION
**Pull request by shams-concentricai**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
The PR is meant to add additional features and expand on file details for the Integration. It adds another step in the Playbook as well.

## Screenshots
image

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ X] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ X] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by shams-concentricai** [comment]
Hi @DeanArbel  
Hope you are doing well !
I am hoping to get this wrapped up quickly , Please let me know if any changes are required !
Looking forward to hear from you.

**Comment by shams-concentricai** [comment]
I am consistently failing the circle validation test. 
i ran the command below 
"demisto-sdk update-release-notes -i Packs/ConcentricAI -u minor" , not sure what it does but even this failed internally ! 
Not sure what is to be done , @DeanArbel  could you please assist !

**Comment by DeanArbel** [comment]
@shams-concentricai The validation is looking for release notes to your changes, and it fails since none were provided.
The suggested way to add release notes is via the `demisto-sdk update-release-notes` command, as it'll create the appropriate release notes file under the `content/Packs/ConcentricAI/ReleaseNotes` folder and bump the pack metadata version. You should run the command with the following arguments:
- `-i` argument with the pack path as its value - this can be either a relative path, or an absolute path.
- `-u` argument with either `minor` or `revision` value - this will determine what'll be the new pack version. e.g. for a pack version that's 1.0.1, the revision will bump it to 1.0.2 and the minor will bump it to 1.1.0.

Do note that after generating the release notes file you'll need to add the exact release notes message manually. 
You'll also need to add the newly created file to your branch via `git add`.

File: Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAI.yml [hunk 1]
```diff
@@ -118,7 +121,18 @@ script:
...
+    - default: false
+      description: File ID
+      isArray: false
+      name: cid
+      required: true
+      secret: false
+    deprecated: false
+    description: Get's file sharing details
+    execution: false
+    name: concentricai-get-file-sharing-details
```
Comment (DeanArbel) [inline Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAI.yml:134]: Please add the command outputs.

File: Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAi_test.py [hunk 2]
```diff
@@ -120,3 +120,14 @@ def test_get_user_details(requests_mock):
     result = get_user_details(loginClient, queryClient, user)
     assert result.outputs_prefix == 'ConcentricAI.UserDetails'
     assert result.outputs_key_field == 'info'
+
+
+def test_get_file_sharing_details(requests_mock):
```
Comment (DeanArbel) [inline Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAi_test.py:125]: Unit tests should contain given-when-then comment to describe the unit test scenario. Given <some input>, When <some state> Then <ensure some behavior>.

**Review by DeanArbel — CHANGES_REQUESTED** [review]
The contribution looks good, but I've got a couple of comments, see below.
Please make sure to revert `conf.json` to origin/master, as the changes that were introduced are causing a conflict.

File: Packs/ConcentricAI/ReleaseNotes/1_1_0.md [hunk 3]
```diff
@@ -0,0 +1,2 @@
+##### ConcentricAI
+- Added File Sharing Details in Playbook along with an additional command.
```
Comment (DeanArbel) [inline Packs/ConcentricAI/ReleaseNotes/1_1_0.md:2]: Your release note doesn't match the expected standard and is missing release notes for the playbook.
I have added the required format below. Please fill in the missing release note for the playbook.
```suggestion
#### Integrations
##### ConcentricAI
- Added File Sharing Details in Playbook along with an additional command.
#### Playbooks
##### ConcentricAI Demo Playbook
```

File: Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAi_test.py (hunk 2 above)
Comment (DeanArbel) [reply Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAi_test.py:136]: @shams-concentricai Please add next to the `given-when-then` comments a short verbal description of what is given (next to **given**), and what's exactly being asserted (next to **then**).

File: Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAI.yml (hunk 1 above)
Comment (shams-concentricai) [reply Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAI.yml:134]: This command results in an array which is represented via Markdown table ! And no outputs are needed for the given command at the moment !

File: Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAi_test.py (hunk 2 above)
Comment (shams-concentricai) [reply Packs/ConcentricAI/Integrations/ConcentricAI/ConcentricAi_test.py:136]: Done !

File: Packs/ConcentricAI/ReleaseNotes/1_1_0.md (hunk 3 above)
Comment (shams-concentricai) [reply Packs/ConcentricAI/ReleaseNotes/1_1_0.md]: Done !

**Comment by shams-concentricai** [comment]
@DeanArbel  I am done with the changes ! I have not added any outputs to the additional command because i am unable to identify what will be the contextPath for it since it returns list of objects. 
Generally it's of the type "contextPath: ConcentricAI.FileInfo.cid" but in case of the additional command will it be of the type "contextPath: ConcentricAI.FileSharingInfo.[0].user_name" ?

**Comment by DeanArbel** [comment]
@shams-concentricai Please add the array output as part of the yaml, but make sure to remove the index path. 

In case of an array result, you should still enter the path without the indexes - e.g. for output like: `ConcentricAI.FileSharingInfo.[0].user_name` the yaml should declare the expected output to be:  `ConcentricAI.FileSharingInfo.user_name`.

By leaving out the indexes, the output will become generic - i.e. users who try to access this context path will  be able to handle either an array output (like shown above) or a single non-array item. If the actual output is an array, the result will be an array of all items in that array - e.g. if we have 2 `user_name` the output will be an array of 2 user_names when trying to access the path: `ConcentricAI.FileSharingInfo.user_name`

If your commands have an array output, please enter that output in the outputs - in case you make any mistakes I'll assist you in correcting them.

**Comment by shams-concentricai** [comment]
Added the outputs now ! Could you please review @DeanArbel

**Review by DeanArbel — APPROVED** [review]

**Comment by DeanArbel** [comment]
Thank you very much for your contribution!
We will create an internal pull request based on your pull request. The internal pull request should be shortly merged after this one.

After it's merged successfully you can expect your contribution to be added to the marketplace.

**Pull request merged**